Weather/Weather/WViewController/ViewController.m

```diff
@@ -0,0 +1,158 @@
+//
+//  ViewController.m
+//  Weather
+//
+//  Created by Sangwook Wi on 5/2/17.
+//  Copyright © 2017 Sangwook Wi. All rights reserved.
+//
+
+#import "ViewController.h"
+#import "WModel.h"
+#import "WCenter.h"
+#import "WTableViewCell.h"
+
+@interface ViewController() {
+    WCenter *cInstance;
+    CGRect originalsize;
+    NSArray *dayModel;
+}
+
+@property(strong, nonatomic) NSMutableArray *daysCell;
+@property(strong, nonatomic) NSMutableArray *highTempCell;
+@property(strong, nonatomic) NSMutableArray *lowTempCell;
+@property(strong, nonatomic) NSMutableArray *weatherCell;
+@property(strong, nonatomic) WModel *cityModel;
+
+@end
+
+@implementation ViewController
+
+bool locationViewOpened;
+
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    locationViewOpened = false;
+    [self unhide];
+    _locationTextField.returnKeyType = UIReturnKeyDone;
+    [_locationTextField setDelegate:self];
+    cInstance = [WCenter sharedInstance];
+    [cInstance fetchModelData:^(WModel *dataModel) {
+        [self update:dataModel];
+    }];
+    
+}
+
+
+- (void)update:(WModel *)data {
+    
+    self.cityModel = data;
+    WDayModel *dayOne = [self.cityModel.days firstObject];
+    self.curWeather.text = dayOne.weather;
+    self.curTemp.text = dayOne.current;
+    self.todayDay.text = dayOne.name;
+    self.todayLow.text = dayOne.low;
+    self.todayHigh.text = dayOne.high;
+    self.city.text = self.cityModel.city;
+    [self setWeatherImage:dayOne.weather forImageview:self.wImage1];
+    [self.tableView reloadData];
+}
+
+-(void)setWeatherImage:(NSString *)weather forImageview:(UIImageView *)imageView {
+    if([weather isEqualToString: @"clear sky"]){
```

Comment:
Minor and nitpick on styling - but keep an eye out on the spacing in your final PRs because it will get called out on when you're working with the Words codebase!